ipaserver/install/cainstance.py

```diff
@@ -1750,27 +1754,68 @@ def import_included_profiles():
     api.Backend.ra_certprofile._read_password()
     api.Backend.ra_certprofile.override_port = 8443
 
-    for (profile_id, desc, store_issued) in dogtag.INCLUDED_PROFILES:
+    for (rule_id, template, helper) in dogtag.INCLUDED_TRANSFORMATION_RULES:
```

Comment:
I'm working on code that will allow profiles with field mapping rules to be imported from files (and at some point will probably do the same for mapping rulesets/transformation rules). This should allow some of the details of the rules to be taken out of the initialization code and moved to profile files instead.

That said, it should be possible to use the API rather than LDAP calls to set up these objects, even without making the other improvement, so I'll take a look at doing that.